desktop/core/src/desktop/static/desktop/js/js.cookie.js

```diff
@@ -0,0 +1,147 @@
+/*!
+ * Javascript Cookie v1.5.1
+ * https://github.com/js-cookie/js-cookie
+ *
+ * Copyright 2006, 2014 Klaus Hartl
+ * Released under the MIT license
+ */
+(function (factory) {
```

Comment:
There's no need for the static file,

If `Cookies` needs to be in the global js scope you can import it and set it on the window object in hue.js.